comments (#68)
aaronpeikert/bayes-prereg: manuscript.Rmd

<!---many researcher do not subscribe to confirmatory agenda--->
We argue that such criticism is not directed against preregistration itself but against a justification through a confirmatory research agenda [@wagenmakersAgendaPurelyConfirmatory2012].
When researchers criticize preregistration as being too inflexible to fit their research question, they often simply acknowledge that their research goals are not strictly confirmatory.
Forcing researchers into adopting a strictly confirmatory research agenda does not only imply changing *how* they investigate a phenomenon but also *what* research questions they ask.
However reasonable such a move is, changing core believes of a large community is much harder than convincing them that a method is well justified.
We therefore attempt to disentangle the *methodological* goals of preregistration from the *ideological* goals of confirmatory science.
It is likely that psychology needs more confirmatory studies to progress as a science, but quite independently of this, preregistration can be useful for any kind of study on the continuum between strictly confirmatory and fully exploratory.

<!---outline-structure--->

> To form such an objective for preregistration, we first introduce some tools of Bayesian philosophy of science and map the exploration/confirmation distinction onto a dimensional quantity we call "theoretical risk" [a term borrowed from @meehlTheoreticalRisksTabular1978, but formalized as the probability of proving a hypothesis wrong, if it does not hold], which is inversely related to the type-I error rate in statistical test theory.

Reviewer: ```suggestion
To form such an objective for preregistration, we first introduce some tools of Bayesian philosophy of science and map the exploration/confirmation distinction onto a dimensional quantity we call "theoretical risk" [a term borrowed from @meehlTheoreticalRisksTabular1978, but formalized as the probability of proving a hypothesis wrong, if it does not hold], which is inversely related to the type-I error rate in null hypothesis testing.
```

We hope to have persuaded the reader to accept two arguments: First, the theoretical risk is important for judging evidential support for theories.
Second, the theoretical risk is inherently uncertain and the degree of uncertainty diminishes the persuasiveness of the gathered evidence.
The third and last argument is that preregistrations reduce this uncertainty.
Following the last argument, a preregistered study is represented by the solid line (certainty about theoretical risk) and a study that was not preregistered is more similar to the dashed line (maximally uncertain about theoretical risk) in Figure \ref{fig:posterior} and Figure \ref{fig:other-measures}.

Let us recall our three assumptions:

1. Researchers judge the evidence for or against a hypothesis rationally.

> 2. They expect other researchers to apply the same rational process.

Reviewer: ```suggestion
2. They expect other researchers to apply a similar rational process.
```

> 2. All else being equal, researchers try to increase the expected epistemic value for other researchers.

Reviewer: ```suggestion
2. Researchers try to maximize the expected epistemic value for other researchers.
```

In that case, the readers can be certain, that they have received a complete account of the situation.
They still might be uncertain about the actual theoretical risk the authors took, but to a much smaller extend than if the study would not have been preregistered.
Remaining sources of uncertainty might be unfamiliarity with statistical methods or experimental paradigms used, the probability of an implementation error in the statistical analyses, a bug in the software used for analyses, etc.
In any case a well written preregistration should aim to reduce the uncertainty about the theoretical risk and hence increase the persuasiveness of evidence.
Therefore, a study that perfectly adhered to its preregistration will resemble the solid line in Figure \ref{fig:posterior}/\ref{fig:other-measures}.
Crucially, perfect means here that the theoretical risk can be judged with low uncertainty, not that the theoretical risk is necessarily high.

# Discussion

To summarize, we showed that both higher theoretical risk and lower uncertainty about theoretical risk lead to higher expected epistemic value across a variety of measures.

> The former result, that increasing theoretical risk leads to higher expected epistemic value, reconstructs the appeal and central goal of preregistration ofr confirmatory research agendas.

Reviewer: ```suggestion
The former result, that increasing theoretical risk leads to higher expected epistemic value reconstructs the appeal and central goal of preregistration of confirmatory research agendas.
```
